src/types/constants.ts

```diff
@@ -2,4 +2,5 @@ export enum TruncateMaxNumber {
   ATTACHMENT = 15,
   SELECTOR = 16,
   CLIENT_TASK_DESCRIPTION = 256,
+  TASK_CARD_NAME = 22,
```

Comment:
I think all of these can be removed and simply use the css styling for truncation. We can create an `hoc` for re-usability. 

I think using such numbers for truncation will bring inconsistency and won't be usable in all kinds of design. It's better to let CSS handle such things for various screen sizes. Styling related jobs should be given to CSS as much as possible for easy life.